corehq/apps/hqmedia/models.py

```diff
@@ -469,6 +468,8 @@ def __init__(self, path,
 
         self.app_lang = app_lang or "en"
 
+        self.languages_linked = languages_linked
+
```

Comment:
where is this used?

Reply:
The `ApplicationMediaReference` gets accessed in the view. This is what allows the front end to know if a particular media item has this value selected or not.